GEON-PPANG-iOS/Presentation/Tabbar/TabBarController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TabBarController.swift
+//  GEON-PPANG-iOS
+//
+//  Created by JEONGEUN KIM on 2023/07/08.
+//
+
+import UIKit
+
+final class TabBarController: UITabBarController {
+
+    // MARK: - Property
+    
+    private var tabs: [UIViewController] = []
+    private lazy var defaultTabBarHeight = { tabBar.frame.size.height }()
+    
+    // MARK: - Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setTabBarItems()
+        setTabBarUI()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        setTabBarHeight()
+    }
+    
+    // MARK: - Setting
+
+    private func setTabBarItems() {
+        tabs = [HomeViewController(),
+                BakeryListViewController(),
+                MyPageViewController()
+                ]
```

Comment:
P5:
그... 저거 줄바꿈.. 해주면 조금 더 편-안할거 같은데...

```suggestion
        tabs = [
            HomeViewController(),
            BakeryListViewController(),
            MyPageViewController()
        ]
```

헤헤...

Reply:
아니면 이렇게 ... ?

```suggestion
        tabs = [HomeViewController(),
                BakeryListViewController(),
                MyPageViewController()]
```

Reply:
첫번째 의견으로 바꾸겠습니다 하하 ㅎㅎ